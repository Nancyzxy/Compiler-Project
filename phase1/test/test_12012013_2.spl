int main()
{
    int a = 0x00a;
    int BBA = a_;
    int _;
    int 3a;
    char c = '\o901';
    if (a0 == 0) _ = b0;
    else _ = -b0;
    while (b != 0) { b1 = b0 - b2; }

    return 0;
}