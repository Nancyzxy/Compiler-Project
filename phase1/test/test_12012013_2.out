Error type A at Line 3: unknown lexeme 0x00a
Error type A at Line 6: unknown lexeme 3a
Error type A at Line 7: unknown lexeme '\o901'
